Controller/Adminhtml/Import/Ajax.php

```diff
@@ -48,19 +48,25 @@ public function __construct(
     public function execute()
     {
         $storeId = (int)$this->request->getParam('storeId');
-
+        $backendParams = $this->helper->buildBackendParams($storeId);
+        $client = new Client($backendParams['time'], $backendParams['token'], $backendParams['platform'], $backendParams['pluginVersion']);
```

Comment:
What is the point of generating the array and then expanding it?

Reply:
BackendParams will be build in the Ajax controller and Data helper function returning them as array is removed.